crates/vhost-user-backend/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Added
 
 ### Changed
+- Change uses of master/slave for frontend/backend in the codebase.
```

Comment:
Maybe we should also point out that the features got renamed? That just tripped me off because `cargo update` silently refuses to update. Only when forcing the version I learn about the reason:

```
❯ cargo update -p vhost@0.8.0 --precise 0.8.1
    Updating crates.io index
error: failed to select a version for `vhost`.
    ... required by package `vhost-device-gpio v0.1.0 (/home/ablu/projects/rust-vmm/vhost-device/crates/vhost-device-gpio)`
versions that meet the requirements `^0.8` are: 0.8.1

the package `vhost-device-gpio` depends on `vhost`, with features: `vhost-user-slave` but `vhost` does not have these features.


failed to select a version for `vhost` which could resolve this conflict
```

Reply:
Yeah, good point. We should also add a link to this PR as we usually do in the changelog

Reply:
I'm going to open a Draft PR for the next release, and I'll include this change there, so we don't forget it!

Reply:
@aesteve-rh don't worry, we need to update the CHANGELOG in any case, so I can handle that in that PR. I'll post soon a draft and ask you and @Ablu to review ;-)

Reply:
@Ablu just to be sure, we need to put that change in the `vhost` changelog, not this one (`vhost-user-backend`), right?

Reply:
> @Ablu just to be sure, we need to put that change in the `vhost` changelog, not this one (`vhost-user-backend`), right?

Right. I find it a bit confusing that there is a backend feature for `vhost` but also a `vhost-user-backend` crate... But I have not digged into what exactly happens within `vhost` there.

Reply:
@Ablu eheh this is another good point, but I don't know how we can fix without a disruptive change :-(

Reply:
> @Ablu eheh this is another good point, but I don't know how we can fix without a disruptive change :-(

Yeah, for now just documenting it in `vhost`'s CHANGELOG should be fine. The other thing is unrelated anyway.

Reply:
Handling this in https://github.com/rust-vmm/vhost/pull/193